dask/dataframe/core.py

```diff
@@ -608,7 +593,7 @@ def rolling(self, window, min_periods=None, win_type=None, axis=0):
                 raise ValueError('min_periods must be >= 0')
 
         return Rolling(self, window=window, min_periods=min_periods,
-                       axis=axis, win_type=win_type)
+                       freq=freq, center=center, win_type=win_type, axis=axis)
```

Comment:
I left in the argument to let us raise `NotImplementedError` rather than `TypeError`. I can remove it entirely if that's the right thing.